src/Workspaces/CoreTest/SemanticModelReuse/SemanticModelReuseTests.cs

```diff
@@ -206,6 +208,28 @@ public async Task MultipleBodyEditsShouldProduceFreshModel_Accessor_Indexer_CSha
             Assert.True(model3.IsSpeculativeSemanticModel);
         }
 
+        [Fact]
+        [WorkItem(1541001, "https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1541001")]
+        [WorkItem(1587699, "https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1587699")]
+        public async Task TestOutOfBoundsInSyntaxContext1_CSharp()
```

Comment:
these tests serve to at least tell us if we've done something bad in the future.  adding one for VB indicated that VB has a similar issue.